incubator/solr/templates/exporter-deployment.yaml

```diff
@@ -0,0 +1,103 @@
+{{- if .Values.exporter.enabled }}
+---
+
+apiVersion: "v1"
+kind: "Service"
+metadata:
+  name: "{{ include "solr.exporter-name" . }}"
+  labels:
+{{ include "solr.common.labels" . | indent 4 }}
+    component: "exporter"
+  annotations:
+{{ toYaml .Values.exporter.service.annotations | indent 4}}
+spec:
+  type: "{{ .Values.exporter.service.type }}"
+  ports:
+    - port: {{ .Values.exporter.port }}
+      name: "solr-client"
+  selector:
+    app: "{{ template "solr.name" . }}"
+    release: "{{ .Release.Name }}"
+    component: "exporter"
+
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "solr.exporter-name" . }}
+  labels:
+{{ include "solr.common.labels" . | indent 4 }}
+    component: "exporter"
+spec:
+  selector:
+    matchLabels:
+      app: "{{ template "solr.name" . }}"
+      release: "{{ .Release.Name }}"
+      component: "exporter"
```

Comment:
For the labels, across this PR, could you please use the new keys. They came from the Kubernetes project... https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata

Reply:
I have updated all the labels to be the new keys.